### ISSUE TITLE
The VoiceOver is not announcing the name of the drop down list box of the Updates preference window of the Xamarin workbooks.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#270]
### Steps to Reproduce

1. Launch Xamarin Workbook application-> Select Mac platform and create a new workbook.
Turn On the VoiceOver.
2. Navigate to Xamarin Workbooks-> preferences->Hit On Enter key-> General tab is opened by default.
3. switch to the Updates window.
4. Hit on Tab key to navigate to the interactive elements of the Updates preference window-> 
5. Hit on Tab key to navigate to "Automatically check for updates:" drop down list box in preference window.
5. Observe the Voiceover announcement for this "Automatically check for updates:" drop down list box. 

### Expected Behaviour
The VoiceOver should announce the name of the drop down list box, so user could understand the purpose of the drop down list box.

### Actual Behaviour
The VoiceOver is not announcing the name of the "Automatically check for updates:" drop down list box.
The VoiceOver is announcing as "At Startup/Hourly/daily/Weekly/Never" popup button, but its not announcing the purpose or name of the drop down list box.

### User impact:
The VoiceOver user won’t be able to know the purpose of the popup button, until the name of the drop down list  box is not being announced
[VO5_AutoCheckUpdate_Preference.zip](https://github.com/Microsoft/workbooks/files/1990877/VO5_AutoCheckUpdate_Preference.zip)
 by the VoiceOver.

### Environment
The VoiceOver user won’t be able to know the purpose of the popup button, until the name of the drop down list  box is not being announced by the VoiceOver.

### Log File

[Uploading VO5_AutoCheckUpdate_Preference.zip…]()

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS4.2.1
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1826
